detox/test/e2e/17.datePicker.test.js

```diff
@@ -1,6 +1,8 @@
 const jestExpect = require('expect').default;
 
-describe.skipIfNewArchOnIOS('DatePicker', () => {
+// todo(new-arch): tests are failing
```

Comment:
So to-do is obsolete now, isn't it?

Reply:
true, removing

Reply:
done: https://github.com/wix/Detox/pull/4704